Error when compiling the layers control with TypeScript 4.8

```
src/control/RLayers.tsx:111:59 - error TS2769: No overload matches this call.
  The last overload gave the following error.
    Argument of type '{ visible: boolean; }' is not assignable to parameter of type 'Partial<unknown> & Attributes'.
      Object literal may only specify known properties, and 'visible' does not exist in type 'Partial<unknown> & Attributes'.

111                         return React.cloneElement(child, {visible: visible[i]});
                                                              ~~~~~~~~~~~~~~~~~~~

  node_modules/@types/react/index.d.ts:320:14
    320     function cloneElement<P>(
                     ~~~~~~~~~~~~
    The last overload is declared here.


Found 1 error in src/control/RLayers.tsx:111
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [1.4.4] WIP
+
+-   Fix #80, error when compiling the layers control with TypeScript 4.8
+
 ### [1.4.3] 2022-08-19
 
 -   Support OpenLayers 7.0.0

--- a/package.json
+++ b/package.json
@@ -94,7 +94,7 @@
     "@types/node": "^17.0.35",
     "@types/prismjs": "^1.16.3",
     "@types/proj4": "^2.5.2",
-    "@types/react": "^18.0.0",
+    "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.5",
     "@typescript-eslint/eslint-plugin": "^5.26.0",
     "@typescript-eslint/parser": "^5.26.0",

--- a/src/control/RLayers.tsx
+++ b/src/control/RLayers.tsx
@@ -3,6 +3,7 @@ import {Control} from 'ol/control';
 
 import {RContextType} from '../context';
 import RControlBase, {RControlOptions, RControlProps} from './RControlBase';
+import {RLayerProps} from 'rlayers/layer/RLayer';
 
 /**
  * @propsfor RLayer
@@ -108,7 +109,7 @@ export default class RLayers extends RControlBase<RLayersProps, RLayersState> {
                 </div>
                 {React.Children.map(this.props.children, (child, i) => {
                     if (React.isValidElement(child)) {
-                        return React.cloneElement(child, {visible: visible[i]});
+                        return React.cloneElement<RLayerProps>(child, {visible: visible[i]});
                     }
                     return child;
                 })}

